### ISSUE TITLE
ChromeIPass Removed from App Store?

### ISSUE THREAD

**Issue by kintaroju** [pfn/keepasshttp#380]
Hi,

I am not sure who maintains chromeipass on the chrome app store, but the link doesn't work anymore, nor can I find it in Chrome Extension searches, except for this one although it differs from the one from this this website's indicated URL to visit.:

One that I found:
https://chrome.google.com/webstore/detail/keepasshttp-connector/dafgdjggglmmknipkhngniifhplpcldb

One that is listed on this website:
https://chrome.google.com/webstore/detail/chromeipass/ompiailgknfdndiefoaoiligalphfdae?hl=en

If someone can place some insight on this that would be great.

Cheers

**Comment by pfn** [comment]
yes, it's currently removed due to new google developer policies.

I have not had any time to maintain any of my projects, and thus they are getting out of date  :(

**Comment by kintaroju** [comment]
Hi,

If that is the case, would that forked version that looks like your product be considered an alternative, or should I stay away from it?

Cheers

**Comment by pfn** [comment]
It's probably OK to use, but it is also in a deprecated state and may go away soon-ish.

**Comment by kintaroju** [comment]
Hi,

Thanks for the info, one last thing, what's the policy for an app to get delisted when it doesn't have active development?

**Comment by Intuii** [comment]
Will this be fixed? If not, what is a suitable alternative for using keepass to log into sites with chrome?

**Comment by zakaluka** [comment]
Here is a project that appears to be maintained and provides similar (but not all) the functionality provided by KeePassHTTP.

https://github.com/subdavis/Tusk .  This is available for Chrome and Firefox (at least), but unfortunately doesn't appear to work on Opera (if you use that browser).

EDIT: Opera integration seems to be fixed in master ( subdavis/Tusk#300 ) but a new release hasn't been cut yet.

**Comment by nifr** [comment]
Following up on the alternative suggestion by @zakaluka it should be noted that as of 2019-07-15 

>  Tusk is no longer actively maintained

([reference](https://github.com/subdavis/Tusk/commit/b528fb0dc50de464991c0a3c39f72687dd7c3c01)).

**Comment by gpion** [comment]
Swicth to Kee instead then
https://chrome.google.com/webstore/detail/kee-password-manager/mmhlniccooihdimnnjhamobppdhaolme

**Comment by alan-null** [comment]
https://github.com/pfn/keepasshttp/issues/425#issue-3605033636
